.jekyll-cache/Jekyll/Cache/Jekyll--Converters--Markdown/c7/71564f874a8bb83404d37c922fd6727a44d1cbcbee928c1d9c82a3c2d52fb6

```diff
@@ -0,0 +1,32 @@
+I"<table>
+  <thead>
+    <tr>
+      <th>Test Instruction</th>
+      <th>Instruction Detail</th>
+      <th>Expected Test Case Result</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td><strong>IC-1</strong></td>
+      <td>Identify any <code class="language-plaintext highlighter-rouge">&lt;img&gt;</code> elements.</td>
+      <td>One <code class="language-plaintext highlighter-rouge">&lt;img&gt;</code> exists in the test data code sample.</td>
+    </tr>
+    <tr>
+      <td><strong>IC-1</strong></td>
```

Comment:
there are two IC-1 identifiers which could be confusing.

Reply:
Yes, I was trying to address to different aspects of the Identify Content instruction, even though the second part is really just assumed in the section (i.e., is the image meaningful?). So, it feels like two separate steps of a test case that require separate validation, even though there is only one numbered Identify Content instruction.

Reply:
Identify Content from Baseline 6.1 is "Identify any image that conveys information (include images of text; functional images used to initiate action, convey meaning, or prompting a response; image maps, etc.)." I don't think it should be modified for the alignment table. This should be copy & paste also.

Reply:
OK, yes, agreed. Fixed.